instrumentation/anr/src/main/java/io/opentelemetry/android/instrumentation/anr/AnrWatcher.kt

```diff
@@ -73,6 +73,7 @@ internal class AnrWatcher(
         }
     }
 
+    @Suppress("DEPRECATION") // suppress deprecation for id
```

Comment:
Should we switch the implementation to use `threadId` instead then?

Reply:
Yeah that sounds reasonable, we'd need to add an API version check as that function was only added in API 36.